src/Mod/AddonManager/addonmanager_workers.py

```diff
@@ -30,6 +30,7 @@
 import FreeCAD
 
 from PySide import QtCore
+import markdown
```

Comment:
I think it's a bad idea to add a new dependency if the module won't be used a lot. This `markdown` module is not part of the standard library, so we would need to package it together with FreeCAD in the AppImage (Conda), or it would need to be marked as a dependency for the .deb package (Debian/Ubuntu).

If the benefits aren't large enough, I don't think we should include this `markdown` module.

Reply:
Markdown is BSD licensed, so I think that extracting required functionality from it into AddonManager is a way to go. I really don't feel comfortable extracting large portions of html via regexp :)

Reply:
If it's BSD licensed, including `markdown` in the `src/3rdParty` directory could be an option. In the past small libraries like that have been added as well.

I suggest you also check the [Extension manager](https://github.com/mnesarco/FreeCAD_ExtMan) workbench that was recently developed. It was started with the intention of improving the current Addon Manager, but now it is essentially a complete replacement. Maybe you can try it and get some ideas from it.